lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -4497,6 +4498,7 @@ ParseResult FIRCircuitParser::parseClass(CircuitOp circuit, unsigned indent) {
   // build it
   auto builder = circuit.getBodyBuilder();
   auto classOp = builder.create<ClassOp>(info.getLoc(), name, portList);
+  SymbolTable::setSymbolVisibility(classOp, SymbolTable::Visibility::Private);
```

Comment:
This could be simplified to:
```suggestion
  classOp.setPrivate();
```
Which seems like a win here but up to you, esp re:what makes sense for other places this is done in this PR re:consistency.

There is `classOp.setVisibility()` if you prefer, FWIW.